probe.go

```diff
@@ -1011,6 +1011,127 @@ func probeCertificates(c FortiHTTP) ([]prometheus.Metric, bool) {
 	return m, true
 }
 
+func probeLoadBalanceServers(c FortiHTTP) ([]prometheus.Metric, bool) {
+	var (
+		virtualServerInfo = prometheus.NewDesc(
+			"fortigate_lb_virtual_server_info",
+			"Info metric regarding virtual servers",
+			[]string{"vdom", "name", "ip", "port", "type"}, nil,
+		)
+		realServerInfo = prometheus.NewDesc(
+			"fortigate_lb_real_server_info",
+			"Info metric regarding real servers",
+			[]string{"vdom", "virtual_server", "id", "ip", "port"}, nil,
+		)
+		realServerMode = prometheus.NewDesc(
+			"fortigate_lb_real_server_mode",
+			"Mode of this real server: active, standby or disabled",
+			[]string{"vdom", "virtual_server", "id", "mode"}, nil,
+		)
+		realServerStatus = prometheus.NewDesc(
+			"fortigate_lb_real_server_status",
+			"Status of this real server: up, down or unknown",
+			[]string{"vdom", "virtual_server", "id", "state"}, nil,
+		)
+		realServerSessions = prometheus.NewDesc(
+			"fortigate_lb_real_server_active_sessions",
+			"Number of sessions active on this real server",
+			[]string{"vdom", "virtual_server", "id"}, nil,
+		)
+		realServerBytesProcessed = prometheus.NewDesc(
+			"fortigate_lb_real_server_processed_bytes_total",
+			"Number of bytes processed by this real server",
+			[]string{"vdom", "virtual_server", "id"}, nil,
+		)
+	)
+
+	type RealServer struct {
+		IP     string `json:"real_server_ip"`
+		Port   int    `json:"real_server_port"`
+		ID     int    `json:"real_server_id"`
+		Mode   string `json:"mode"`
+		Status string `json:"status"`
+		// MonitorEvents  int    `json:"monitor_events"`
+		ActiveSessions int `json:"active_sessions"`
+		// How to 'parse' this field (e.g. '<1' and expose it as a usable metric?)
```

Comment:
No idea about possible values and unit to be honest, I can't really find a definite format for this field. You might be right that they only use '<1' and otherwise a ms value. Maybe we should just go with that assumption, if/when someone spots a different 'value' in the wild we can add it.

Reply:
@bluecmd can you look into this with your fortigate instance? 

If we can not find any reference this will probably be the best.     
A parsing failure should not lead to other missing metrics and should be definitely logged as an error.